src/Sarif.Viewer.VisualStudio/Themes/CallTrees.xaml

```diff
@@ -31,22 +31,23 @@
                             <cvt:ImportanceToBackgroundConverter x:Key="importanceToBackgroundConverter" />
                             <cvt:ImportanceToEssentialMarkerVisibilityConverter x:Key="importanceToEssentialMarkerVisibilityConverter" />
                         </TreeView.Resources>
+                        <TreeView.ItemContainerStyle>
+                            <Style>
+                                <Style.Triggers>
+                                    <DataTrigger Binding="{Binding Location.Importance}"
+                                                 Value="Essential">
+                                        <Setter Property="TreeViewItem.IsExpanded"
+                                                Value="True" />
+                                    </DataTrigger>
+                                </Style.Triggers>
+                            </Style>
+                        </TreeView.ItemContainerStyle>
+
                         <TreeView.ItemTemplate>
                             <HierarchicalDataTemplate DataType="x:Type local:CallTreeNode" ItemsSource="{Binding Children}">
-                                <StackPanel Orientation="Horizontal">
-                                    <TextBlock Text="! " FontWeight="Bold" Foreground="Red"
-                                               Style="{StaticResource callTreeTextStyle}"
-                                               Background="{Binding Location.Importance, 
-                                                            Converter={StaticResource importanceToBackgroundConverter}}"
-                                               Visibility="{Binding Location.Importance, 
```

Comment:
If you're going to remove the background coloring, then remove the `importanceToBackgroundConverter` resource, the `ImportanceToBackgroundConverter` class, and its unit tests.